api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -204,11 +205,11 @@ public void setUserService(UserService userService) {
     );
     String workspacePath = body.getWorkspaceId();
     if (!projectName.equals(body.getWorkspaceNamespace())) {
-      workspacePath = body.getWorkspaceNamespace() + ":" + body.getWorkspaceId();
+      workspacePath = body.getWorkspaceNamespace() + WORKSPACE_DELIMITER + body.getWorkspaceId();
     }
     String apiDir = "workspaces/" + workspacePath;
     if (body.getPlaygroundMode()) {
-      apiDir = "workspaces:playground/" + workspacePath;
+      apiDir = "workspaces" + WORKSPACE_DELIMITER + "playground/" + workspacePath;
```

Comment:
I'd name the constant more generically, or else just hardcode _ here. `workspaces:playground` doesn't have any of the same uniqueness constraints as the other directory. Could also just be a single underscore in this case.